docs/source/reference/cli.rst

```diff
@@ -28,6 +28,19 @@ page.
     -s STYLE_FILE, --style-file STYLE_FILE
                             colour bar definition to use
 
+
+The ``cset bake`` commands need to be run out of the ``CSET/recipe`` folder.
+Here is an example to run a recipe making use of the templated variable ``VARNAME`` in the recipe.
+The '-vv' is optional to give very verbose output::
+
+.. code-block:: text
+
+    cset -vv bake -i INPUT_DIR/* -o OUTPUT_DIR -r generic_surface_histogram_series.yaml  --VARNAME air_pressure_at_sea_level --VALIDITY_TIME 2024-01-16T06:00Z
```

Comment:
```suggestion
The '-v' is optional to give verbose output::

.. code-block:: text

    cset -v bake -i INPUT_DIR/* -o OUTPUT_DIR -r generic_surface_histogram_series.yaml  --VARNAME air_pressure_at_sea_level --VALIDITY_TIME 2024-01-16T06:00Z
```
The `-vv` option prints out all of the debugging info, which is a lot more than most people want. The info level debugging (`-v`) should be our recommendation for most people.